spec/person_spec.rb

```diff
@@ -0,0 +1,31 @@
+require_relative '../modules/person'
+
+RSpec.describe Person do
```

Comment:
- [ ] Hello team! 👋 In the Person class, we've noticed that the add_rental method and correct_name method are currently not covered by any tests. 🚨 Let's make sure to add test coverage for this method. Additionally, please double-check all other methods in the Person class to ensure that they are also covered by RSpec tests. 🕵️‍♂️ Let's make sure our code is thoroughly tested and reliable! 🧪💻